test/code_size/test_codesize_hello_dylink.json

```diff
@@ -1,10 +1,10 @@
 {
-  "a.out.js": 26969,
-  "a.out.js.gz": 11479,
+  "a.out.js": 26998,
```

Comment:
Not that it's a big deal but why are both the JS and wasm getting bigger here?

Reply:
I think the JS is getting bigger just because of that extra `&& typeof value.value != 'undefined'` check, but i didn't actually confirm.   Since this is such as small fraction of the JS size for dynamc linking I'm not sure its worth pin pointing it.